lib/spans/span-context.js

```diff
@@ -1,13 +1,36 @@
 'use strict'
 
+const {MAXIMUM_CUSTOM_ATTRIBUTES} = require('../attributes')
+const {PrioritizedAttributes, ATTRIBUTE_PRIORITY} = require('../prioritized-attributes')
+const {DESTINATIONS} = require('../config/attribute-filter')
+
+// Scoping impacts memoization. We could decide to add a scope instead of including
+// spans in segment scope in the future.
+const ATTRIBUTE_SCOPE = 'segment'
+
 class SpanContext {
-  constructor(intrinsicAttributes) {
+  constructor(intrinsicAttributes, customAttributes) {
     this.intrinsicAttributes = intrinsicAttributes || Object.create(null)
+
```

Comment:
And adding custom attributes to the span context. But using Prioritized attributes.